### ISSUE TITLE
Remove advanced installation methods

### ISSUE THREAD

**Issue by SunSince90** [CloudNativeSDWAN/cnwan-operator#46]
Advanced installation is mostly useless: most people that want to contribute already know how to do that as they probably know Kubernetes and that the simple installation method is actually way more than enough.

Its existence is basically only a burden, as it is just one more thing to support and update while being rarely used.

**Comment by ljakab** [comment]
@SunSince90 , I wonder if it's possible (and makes sense) to remove or generalize the parts that need the most maintenance, and maybe rename the advanced installation guide to contributor guide, and refocus it on how one could/should contribute to the project?

**Comment by SunSince90** [comment]
Yes i thought doing something like that, dropping all the  files and methods included in the advanced installation guide and just write a guide on how to contribute, including the files that need to be changed in order to test the contribution.
